docs/api/ref/GetAgent.md

```diff
@@ -0,0 +1,224 @@
+---
+id: GetAgent
+sidebar_label: GetAgent
+title: GetAgent
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+The `getAgent()` function is the recommended way for **web applications** to connect to an FDC3 Desktop Agent. The function is exported by the [`@finos/fdc3`](https://www.npmjs.com/package/@finos/fdc3) NPM module and returns a [Desktop Agent](./DesktopAgent) implementation:
+
+<Tabs groupId="lang">
+<TabItem value="ts" label="TypeScript">
+
+```ts
+import { getAgent, DesktopAgent } from "@finos/fdc3";
+
+try {
+    const desktopAgent: DesktopAgent = await getAgent();
+    //do FDC3 things here
+} catch (e: AgentError) {
+    // Failed to connect
+}
+```
+
+</TabItem>
+<TabItem value="js" label="JavaScript">
+
+```js
+import { getAgent } from "@finos/fdc3";
+
+try {
+    const desktopAgent = await getAgent();
+    //do FDC3 things here
+} catch (e) {
+    // Failed to connect
+}
+```
+
+</TabItem>
+</Tabs>
+
+The  `getAgent()` function allows web applications to retrieve an FDC3 Desktop Agent API interface to work with, whether they are running in an environment that supports a Desktop Agent Preload (a container-injected API implementation) or a Desktop Agent Proxy (a Browser-based Desktop Agent running in another window or frame). The behavior of `getAgent()` is defined by the [FDC3 Web Connection Protocol (WCP)](../specs/webConnectionProtocol) and communication with a Desktop Agent Proxy in a web-browser is defined by the [Desktop Agent Communication Protocol (DACP)](../specs/desktopAgentCommunicationProtocol). Hence, it allows applications to be written that will work in either scenario without modification or the inclusion of vendor-specific libraries.
+
+If no Desktop Agent is found, a failover function may be supplied by app allowing it to start or otherwise connect to a Desktop Agent (e.g. by loading a proprietary adaptor that returns a `DesktopAgent` implementation or by creating a window or iframe of its own that will provide a Desktop Agent Proxy).
```

Comment:
Consider changing "If no Desktop Agent is found" to something like "To handle the scenario where Desktop Agent can be found"